pages/common/ned.md

```diff
@@ -0,0 +1,35 @@
+# ned
+
+> Is like grep, but with powerful replace capabilities, and unlike sed, as it isn't restricted to line oriented editing.
+
+- Recursively search starting in the current directory, ignoring case:
+
+`ned -iR '^[dl]og' .`
```

Comment:
Can the user change `^[dl]og`? If so, could we wrap it in `{{` and `}}` like this please? Same with `.` for the current directory.

```suggestion
`ned -iR '{{^[dl]og}}' {{.}}`
```

This is because `{{` and `}}` is our custom markdown extension that indicates that the user can change something - and is handled specially by clients.

Reply:
Done.